.secrets.baseline

```diff
@@ -41,7 +41,7 @@
     },
     {
       "name": "KeywordDetector",
-      "keyword_exclude": ""
+      "keyword_exclude": "secretsmanager|privateKeyFile|ARTIFACT_SOURCE_BUCKET_NAME_SECRET|GH_ACTIONS_ROLE_ARN_SECRET"
```

Comment:
I liked this approach, but I didn't see anything that allowed us to specify these values in a config file - so they're difficult to retain on the next usage of `detect-secrets scan > .secrets.baseline`

Reply:
The command should be run as `detect-secrets scan --baseline .secrets.baseline` - this is how pre-commit does it and it tells the `detect-secrets` utility to use the config from the baseline file - then all those keywords and other exclusions are retained.

Reply:
Ahhhh. We've been using it wrong! 

Can you update the `README.md` to mention that - it'll stop misuse in the future.

Reply:
Why don't we drop using the word secret, why do we need this ... what is secretsmanager ? we don't have one

Reply:
Done ✅

Reply:
@mrdoodles these are the names of various variables/params that are in our GitHub Actions workflows and the SAM templates - they are not secret values and they're already public in this repo.

Reply:
I agree `ARTIFACT_SOURCE_BUCKET_NAME_SECRET ` is a bad param name but updating secret names is not in scope of this PR